cmd/kops/toolbox.go

```diff
@@ -18,14 +18,18 @@ package main
 
 import (
 	"github.com/spf13/cobra"
+	"io"
+	"k8s.io/kops/cmd/kops/util"
 )
 
-// toolboxCmd represents the toolbox command
-var toolboxCmd = &cobra.Command{
-	Use:   "toolbox",
-	Short: "Misc infrequently used commands",
-}
+func NewCmdToolbox(f *util.Factory, out io.Writer) *cobra.Command {
```

Comment:
Don't love the name - but you guys know I am a fan of concrete verbose names over misc naming.. 

Either way, I think we should have a decent chunk of documentation around this. Especially if we are dropping the cluster dump into a `toolbox` command